scripts/tools/extract-tools-github.ts

```diff
@@ -4,7 +4,7 @@ import dotenv from 'dotenv';
 
 import type { ToolsData } from '@/types/scripts/tools';
 
-import { pause } from '../utils';
+import { pause } from '../utils/functions';
```

Comment:
I've combined these utility functions in one file, do update it after I merge my PR.